package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "client": "cd client && npm start",
-    "server": "cd server && npm start",
+    "client": "cd client && npm install && npm start",
+    "server": "cd server && npm install && npm start",
     "dev": "concurrently --kill-others-on-fail \"npm run server\" \"npm run client\"",
```

Comment:
האם אנחנו רוצים לשקול להפוך את זה פשוט לstart?

Reply:
את מה?

Reply:
פשוט רציתי שתהיה הפרדה ברגע שנתחיל להריץ גם בפרודקשן אז שיהיה סקריפט גם לפרודקשן ולפיתוח

Reply:
אתה חושב שהם יהיו הרצות שונות?

Reply:
It has to be 2 running formats. One for our dev environment as developers and one for production to use in the CI-CD pipeline.

English and hebrew together sucks.

Reply:
כפרה

Reply:
אוקיוש קול